tests/requirements/requirements_base.txt

```diff
@@ -1,15 +1,19 @@
 beautifulsoup4
 coverage
 django-app-helper
+django-classy-tags
+django-sekizai
 factory_boy
 flake8
 isort
 lxml
 tox
-https://github.com/django-cms/django-cms/tarball/4.0.0#egg=django-cms
-https://github.com/divio/djangocms-text-ckeditor/tarball/support/4.0.x#egg=djangocms-text-ckeditor
-https://github.com/django-cms/djangocms-versioning/tarball/1.2.2#egg=djangocms-versioning
-https://github.com/FidelityInternational/djangocms-version-locking/tarball/master#egg=djangocms-version-locking
-https://github.com/django-cms/djangocms-moderation/tarball/master#egg=djangocms-moderation
-https://github.com/FidelityInternational/djangocms-references/tarball/1.4.3#egg=djangocms-references
-https://github.com/django-cms/djangocms-alias/tarball/1.11.0#egg=djangocms-alias
+
+# Unreleased django-cms 4.0 compatible packages
+https://github.com/django-cms/django-cms/tarball/release/4.0.1.x#egg=django-cms
+https://github.com/divio/djangocms-text-ckeditor/tarball/master#egg=djangocms-text-ckeditor
+https://github.com/django-cms/djangocms-versioning/tarball/support/django-cms-4.0.x#egg=djangocms-versioning
+https://github.com/joshyu/djangocms-version-locking/tarball/feat/django-42-compatible#egg=djangocms-version-locking
+https://github.com/FidelityInternational/djangocms-moderation/tarball/feature/django-42-compat#egg=djangocms-moderation
```

Comment:
replace `djangocms-moderation` to use  https://github.com/django-cms/djangocms-moderation/tarball/master#egg=djangocms-moderation